$MirOS: ports/www/firesomething/patches/patch-nsprpub_pr_include_private_primpl_h,v 1.5 2008/12/05 16:26:10 bsiegert Exp $
$OpenBSD: patch-nsprpub_pr_include_private_primpl_h,v 1.2 2004/11/26 18:29:15 pvalchev Exp $
--- mozilla/nsprpub/pr/include/private/primpl.h.orig	2008-01-20 05:53:09.000000000 +0100
+++ mozilla/nsprpub/pr/include/private/primpl.h	2008-08-24 14:52:51.000000000 +0200
@@ -1880,6 +1880,7 @@ extern void _PR_DestroyZones(void);
         && !defined(DARWIN) \
         && !defined(NEXTSTEP) \
         && !defined(QNX) \
+        && !defined(__OpenBSD__) \
         && !(defined (UNIXWARE) && defined (USE_SVR4_THREADS))
 #define _PR_OVERRIDE_MALLOC
 #endif
